mbed_tools_ci_scripts/spdx_report/spdx_file.py

```diff
@@ -116,8 +117,8 @@ def generate_spdx_file(self) -> File:
         source_file.type = FileType.SOURCE
         source_file.comment = determine_spdx_value(None)
         source_file.chk_sum = Algorithm("SHA1", self.sha1_check_sum)
-        source_file.conc_lics = License.from_identifier(str(determine_spdx_value(self.licence)))
+        source_file.conc_lics = License.from_identifier(str(determine_spdx_value(self.licence.identifier)))
```

Comment:
I think this attribute could do with a better name. I'm not sure what a `conc_lics` is.

Reply:
this is a field defined by the SPDX SDK. I don't have control over the names